src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -274,7 +274,8 @@ public GHAsset uploadAsset(String filename, InputStream stream, String contentTy
         Requester builder = owner.root().createRequest().method("POST");
         String url = getUploadUrl();
         // strip the helpful garbage from the url
-        url = url.substring(0, url.indexOf('{'));
+        int endIndex = url.indexOf('{');
+        if (endIndex != -1) url = url.substring(0, endIndex);
```

Comment:
We use curly braces in all cases.
```suggestion
        int endIndex = url.indexOf('{');
        if (endIndex != -1) {
            url = url.substring(0, endIndex);
        }
```